meson.build

```diff
@@ -326,6 +326,19 @@ if not features['build-date']
 endif
 
 features += {'ta-leak-report': get_option('ta-leak-report')}
+features += {'modular-drivers': get_option('modular-drivers')}
+if features['modular-drivers'] and not posix
+    error('modular drivers are only supported on posix')
+endif
+moduledir = get_option('libdir') / 'mpv'
```

Comment:
Instead of being a boolean, I think it would better for `modular-drivers` to be a feature type and then you can use `.require` like with other similar checks since this isn't just a simple true/false type of thing.

Reply:
Done, thanks!